scripts/ValidateCode.sh

```diff
@@ -15,23 +15,26 @@ docker-compose exec -T ${CONTAINER_NAME} python -c "import requests; f=open('${C
 [ -d "/mnt/samba/${APP_NAME}" ] || mkdir -p "/mnt/samba/${APP_NAME}"
 docker cp "$(docker-compose ps -q ${CONTAINER_NAME})":"/python/app/${CLEAN_BRANCH_NAME}_openapi.json" "/mnt/samba/${APP_NAME}/${CLEAN_BRANCH_NAME}_openapi.json"
 
-echo "Clean up old reports" 
-rm -f unittesting.xml coverage.xml typing.xml typing-server.xml typing-integrations.xml
+if [[ -n $DEV_ARG ]] 
+then
```

Comment:
issue: A better way to do this is to exit early. Instead of having to scroll down to check what happens after this if statement, I then know that everything else below is for DEV only since otherwise we exit the script entirely.
(Check the code below, my bash knowledge is rusty.

```suggestion
if [[ -z $DEV_ARG ]] 
then
    exit 0
```

Reply:
It works